cypress/integration/ete/reset_password.spec.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types='cypress' />
+
+describe('Password reset flow', () => {
+  context('Account exists', () => {
+    // This test depends on this Mailslurp account already being registered
+    const existing = {
+      email: '0298a96c-1028-4e3d-b943-a2b478c84dbd@mailslurp.com',
+      inbox: '0298a96c-1028-4e3d-b943-a2b478c84dbd',
+    };
+
+    it("changes the reader's password", () => {
+      cy.emptyInbox(existing.inbox).then(() => {
+        cy.intercept({
+          method: 'GET',
+          url: 'https://api.pwnedpasswords.com/range/*',
```

Comment:
general q about behaviour of this api endpoint in the app for my benefit -- is it used to stop people from changing their password to a compromised one?

Reply:
and-if it is-do we want a test case for when this happens?

Reply:
> general q about behaviour of this api endpoint in the app for my benefit -- is it used to stop people from changing their password to a compromised one?

Yes, exactly that. You can play around with this feature (and all of our pages and components) here: https://www.chromatic.com/component?appId=60929d168594f80039336501&name=Pages%2FChangePassword&buildNumber=631

Reply:
> and-if it is-do we want a test case for when this happens?

Excellent point. This is defo something we want to be testing for, but actually [we already do](https://github.com/guardian/gateway/blob/main/cypress/integration/mocked/change_password.spec.js#L103)

Reply:
Yeah, Chromatic is fantastic. It's caught a _lot_ of issues, especially over on DCR.

Whenever you add a story it is automatically captured by Chromatic and then visual regression gets run on each push

Reply:
visual regression checking is super neat, does a regression fail the build or is it mainly there for a quick check?

Reply:
It fails the build if it finds diffs. Someone has to click through to Chromatic and manually approve them (or fix the bug which caused them)

Reply:
nice – sounds like a good setup :)